### PR TITLE
DEVX-1844: examples repo: replace KSQL with ksqlDB

### PR DESCRIPTION
**Pull request by ybyzek**

### CODE REVIEW

File: clients/cloud/ksql-datagen/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Overview
 
-Produce messages to and consume messages from a Kafka cluster using the [ksqlDB datagen](https://docs.confluent.io/current/ksql/docs/tutorials/generate-custom-test-data.html?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.clients-ccloud) command-line tool.
+Produce messages to and consume messages from a Kafka cluster using the [KSQL datagen](https://docs.confluent.io/current/ksql/docs/tutorials/generate-custom-test-data.html?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.clients-ccloud) command-line tool.
```
Comment (vsyu) [inline clients/cloud/ksql-datagen/README.md:3]: Should this still be "ksqlDB"?

**Review by vsyu — APPROVED** [review]
Hi Yeva, I had one question but otherwise everything LGTM!

File: clients/cloud/ksql-datagen/README.md (hunk 1 above)
Comment (ybyzek) [reply clients/cloud/ksql-datagen/README.md:3]: Eng/PM said the datagen stays as "KSQL Datagen"

**Comment by ybyzek** [comment]
@rspurgeon @gianlucanatali note changes to ccloud_library.sh

**Comment by rspurgeon** [comment]
@ybyzek This touches a lot of individual demos. Are you looking for validation that each demo functions appropriately after these wide change?

**Comment by ybyzek** [comment]
> Are you looking for validation that each demo functions appropriately after these wide change?

No, just validate spirit of the change ;)

**Review by rspurgeon — APPROVED** [review]
spirit LGTM

**Comment by gianlucanatali** [comment]
Hi @ybyzek , on the ccloud_library.sh LGTM. I just tested it with my 2 demos that use this lib, but those 2 work well :)

**Review by gianlucanatali — APPROVED** [review]
LGTM

**Pull request merged**